### ISSUE TITLE
Room Leaving Issue

### ISSUE THREAD

**Issue by Simashax** [haxball/haxball-issues#155]
Hello. I don't know if it had been already suggested but nevertheless there is something to be fixed. A player should leave the room instantly not only when he presses the button "leave" but also when he just closes the browser or turns off the PC. It is especially important during official matches because in case of internet/power outage or browser failure the player is still in the field for 30 seconds and others at first glance might not notice him having problems, which can lead to losing match time and other misunderstandings. Thank you

**Comment by Leo-MapCreator** [comment]
I agree on that, but i think 30s is a good time to kick someone, instead than 2 min, HTML5 is way faster than Flash version on removing of the room people that left it
